README.md

```diff
@@ -78,23 +86,127 @@ or without an inventory file,
 
 ```bash
-bolt task run servicenow_tasks::create_ci 
---targets 'dev99218.service-now.com' 
-certname=puppet-master 
+bolt task run servicenow_tasks::create_ci
+--targets 'dev99218.service-now.com'
+certname=puppet-master
 fact_query_results="[{\"name\":\"fqdn\",\"value\":\"puppet-master.c.splunk-275519.internal\"},{\"name\":\"domain\",\"value\":\"c.splunk-275519.internal\"},{\"name\":\"is_virtual\",\"value\":true},{\"name\":\"macaddress\",\"value\":\"42:01:0a:8a:00:03\"},{\"name\":\"processors\",\"value\":{\"isa\":\"x86_64\",\"count\":2,\"models\":[\"Intel(R) Xeon(R) CPU @ 2.20GHz\",\"Intel(R) Xeon(R) CPU @ 2.20GHz\"],\"physicalcount\":1}},{\"name\":\"serialnumber\",\"value\":\"GoogleCloud-F48713898D3A1DF97AF4AFC761243E4C\"},{\"name\":\"memorysize_mb\",\"value\":7812.03515625},{\"name\":\"processorcount\",\"value\":2},{\"name\":\"operatingsystemrelease\",\"value\":\"8.1.1911\"},{\"name\":\"physicalprocessorcount\",\"value\":1}]"
 fact_map="{\"fqdn\":\"fqdn\",\"domain\":\"dns_domain\",\"serialnumber\":\"serial_number\",\"operatingsystemrelease\":\"os_version\",\"physicalprocessorcount\":\"cpu_count\",\"processorcount\":\"cpu_core_count\",\"processors.models.0\":\"cpu_type\",\"memorysize_mb\":\"ram\",\"is_virtual\":\"virtual\",\"macaddress\":\"mac_address\"}"
 ```
 Further example usage is provided via `bolt task show`
 
+```bash
+puppet task run servicenow_tasks::create_ci \
+--nodes sauterne-victim.delivery.puppetlabs.net \
+certname='sauterne-victim.delivery.puppetlabs.net' \
+user='admin' password='74rByafftSFQVCYf' instance='dev106575.service-now.com' \
+fact_query_results="[{\"name\":\"fqdn\",\"value\":\"puppet-master.c.splunk-275519.internal\"},{\"name\":\"domain\",\"value\":\"c.splunk-275519.internal\"},{\"name\":\"is_virtual\",\"value\":true},{\"name\":\"macaddress\",\"value\":\"42:01:0a:8a:00:03\"},{\"name\":\"processors\",\"value\":{\"isa\":\"x86_64\",\"count\":2,\"models\":[\"Intel(R) Xeon(R) CPU @ 2.20GHz\",\"Intel(R) Xeon(R) CPU @ 2.20GHz\"],\"physicalcount\":1}},{\"name\":\"serialnumber\",\"value\":\"GoogleCloud-F48713898D3A1DF97AF4AFC761243E4C\"},{\"name\":\"memorysize_mb\",\"value\":7812.03515625},{\"name\":\"processorcount\",\"value\":2},{\"name\":\"operatingsystemrelease\",\"value\":\"8.1.1911\"},{\"name\":\"physicalprocessorcount\",\"value\":1}]"
+```
+To use oauth_token, replace user and password parameters with oauth_token as shown in create_ci.
+
+### create_record
+This task creates a new record in ServiceNow. You must provide the following parameters in addition to the inventory/authentication data described [above](##setup). The create_record task will return the sys_id of the new record in ServiceNow.
+
+Required Parameters:
+- `table` : ServiceNow table.
+- `fields` : The ServiceNow fields. This is a JSON hash.
+
+```bash
+puppet task run servicenow_tasks::create_record \
+--nodes sauterne-victim.delivery.puppetlabs.net \
+table='incidents' \
+fields="{}" \
```

Comment:
Will this actually create a blank incident?

Reply:
Yes - blank incident with defaulted values